drivers/drivers.go

```diff
@@ -75,6 +77,7 @@ func getSegmentDataHTTP(uri string) ([]byte, error) {
 		glog.Errorf("Error reading body uri=%s err=%v", uri, err)
 		return nil, err
 	}
-	glog.V(common.VERBOSE).Infof("Downloaded uri=%s", uri)
+	took := time.Since(started)
+	glog.V(common.VERBOSE).Infof("Downloaded uri=%s downdur=%s", uri, took)
```

Comment:
Can we maybe use `dur=` as a consistent key for the various forms of duration? (If someone needs to search specifically for the download duration, they can grep for "Downloaded uri=" and so forth)

Reply:
changed